ApollosProject/apollos-apps
# Add user settings route

## ✏️ Description
This PR creates the route to allow direct navigation to the usersettings page. This will be used for new notifications for follow requests so people get sent to where they can see them. 

## 🔬 To Test
1. pull this pr
2. open safari in the sim. You can then put in the deep link with the route /usersettings this should open the app to the right page.
